sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -193,10 +201,28 @@ private RuntimeException logException(RuntimeException runtimeException, int log
      * @param format formattable message.
      * @param args Arguments for the message, if an exception is being logged last argument is the throwable.
      */
-    private void performLogging(int logLevel, String format, Object... args) {
+    private void performLogging(int logLevel, int environmentLogLevel, boolean isExceptionLogging, String format,
```

Comment:
`logLevel` should be sufficient to check whether throwable should be stripped out or not. Don't need the `environmentLogLevel` here.

Reply:
Based on the specification `DO log exceptions thrown as a Warning level message. If the log level set to Verbose, append stack trace information to the message.` I took it to mean if an exception is being logged log it as a warning and if the environment logging level is set to verbose additionally include the stack trace.